src/modules/commander/failsafe/failsafe.cpp

```diff
@@ -452,8 +452,12 @@ void Failsafe::checkStateAndMode(const hrt_abstime &time_us, const State &state,
 
 	// Do not enter failsafe while doing a vtol takeoff after the vehicle has started a transition and before it reaches the loiter
 	// altitude. The vtol takeoff navigaton mode will set mission_finished to true as soon as the loiter is established
-	const bool ignore_link_failsafe = state.user_intended_mode == vehicle_status_s::NAVIGATION_STATE_AUTO_VTOL_TAKEOFF
-					  && in_forward_flight && !state.mission_finished;
+	const bool ignore_any_link_loss_vtol_takeoff_fixedwing = state.user_intended_mode ==
+			vehicle_status_s::NAVIGATION_STATE_AUTO_VTOL_TAKEOFF
+			&& in_forward_flight && !state.mission_finished;
+	const bool ignore_any_link_loss_during_land = state.user_intended_mode == vehicle_status_s::NAVIGATION_STATE_AUTO_LAND
```

Comment:
Isn't it dangerous to ignore any link loss during land? I would prefer if my drone stops descending if I lose manual control ability and then does the failsafe behavior.

Reply:
I agree, that was actually a fix we did since failsafe during takeoff and land originally used to be pretty limited which looked like a corner case loophole.

See
> I also realized that there are no reactions in takeoff mode for link loss. This could potentially result in a flyaway with an unreachable takeoff altitude with lost link.
from the description of the original COM_RCL_EXCEPT pr: https://github.com/PX4/PX4-Autopilot/pull/17785

Reply:
Anyways this was fixed in 1e6b8fe